sysutils/freeradius/Portfile

```
@@ -27,24 +31,24 @@ long_description        The FreeRADIUS Server Project is an attempt to create \
                         RADIUS server, but they don't share a lot in common \
                         anymore.
 
-master_sites            ftp://ftp.freeradius.org/pub/freeradius/ \
-                        ftp://ftp.freeradius.org/pub/freeradius/old
+openssl.branch          1.1
 
-depends_lib             port:libtool \
+depends_lib-append      port:libtool \
                         port:libpcap \
                         path:lib/libldap.dylib:openldap \
-                        path:lib/libssl.dylib:openssl \
                         port:talloc \
-                        port:gdbm
-
-use_autoconf            yes
+                        port:gdbm   \
+                        port:pcre   \
+                        port:perl${perl5.default_branch}
 
-configure.args          --with-openssl-includes=${prefix}/include/openssl \
-                        --with-openssl-libraries=${prefix}/lib \
+configure.args          --with-openssl-includes=[openssl::include_dir] \
+                        --with-openssl-libraries=[openssl::lib_dir] \
                         --with-system-libltdl \
                         --with-system-libtool \
                         --without-rlm_krb5
```

The CI builds still show ${prefix}/bin/perl usage during configure:
```
  checking for perl... /opt/local/bin/perl
```

The version of ${prefix}/bin/perl version can be controlled by variants, so it may be preferable to avoid it in favor of a specific ${prefix}/bin/perl5.xx:

```suggestion
                        --without-rlm_krb5 \
                        --with-perl=${perl5.bin} \
                        PERL=${perl5.bin}
```